src/components/features/PromoBox/PromoBoxContainer.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+
+import PromoBox from './PromoBox';
+
+import { getNew } from '../../../redux/productsRedux.js';
+import { getAll } from '../../../redux/promoImagesRedux.js';
+
+const mapStateToProps = state => ({
+  products: getNew(state),
```

Comment:
Raczej nie nowe produkty tylko na promocji, mamy pole `promo`

Reply:
Czyli rozumiem że mamy mieć np produkty które będą miały wartość promo: 'sale', i wtedy idą do boxa po lewej i produkty o wartości promo: 'JakaśtamWartość' i wtedy wędrują do boksa po prawej, zgadza się?

Reply:
Ok, przerzuciłem te selektory do productsRedux, stamtąd importuje to do containera i następnie w formie propsów przerzucam do komponentu.